plugin/storage/cassandra/spanstore/operation_names_test.go

```diff
@@ -30,6 +30,12 @@ import (
 	"github.com/jaegertracing/jaeger/pkg/testutils"
 )
 
+type test struct {
+	ttl           time.Duration
+	schemaVersion int
+	expErr        error
+}
```

Comment:
this is defined waaaay to far from the two places where it's used. We normally use inline type definition
```
tests := []struct {
	ttl           time.Duration
	schemaVersion int
	expErr        error
}{
   { ... test 1 params ... },
   { ... test 2 params ... },
}
```